azure-pipelines-official.yml

```diff
@@ -116,6 +116,11 @@ stages:
       displayName: Build
       condition: succeeded()
 
+    # Authenticate with service connections to be able to publish packages to external nuget feeds.
+    - task: NuGetAuthenticate@0
+      inputs:
+        nuGetServiceConnections: azure-public/vs-impl, azure-public/vssdk
```

Comment:
Perhaps we should do this before we start build, so that if there is a problem with auth we will see right away rather then only after the build finishes.